modules/islandora_iiif/src/Plugin/views/style/IIIFManifest.php

```diff
@@ -143,6 +154,11 @@ public function render() {
       $content_path = implode('/', $url_components);
       $iiif_base_id = $request_host . '/' . $content_path;
 
+      /**
+       * @var \Drupal\taxonomy\TermInterface|null
+       */
+      $structured_text_term = $this->utils->getTermForUri($this->options['structured_text_term_uri']);
```

Comment:
Would it make more sense to roll a helper method around this, instead of grabbing it here and passing it everywhere (something like https://github.com/Islandora/islandora/compare/hocr_media_redux...adam-vessey:islandora:alternate-term-access)? Or possibly grabbing it in the constructor, to have it available in whatever method(s) needs it?

Reply:
Good point, done.